src/Frontend/Modules/FormBuilder/EventListener/FormBuilderSubmittedMailSubscriber.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Frontend\Modules\FormBuilder\EventListener;
+
+use Swift_Mailer;
+use Frontend\Core\Engine\Language as FL;
+use Frontend\Core\Engine\Model as FrontendModel;
+use Frontend\Modules\FormBuilder\Event\FormBuilderSubmittedEvent;
+
+/**
+ * A Formbuilder submitted event subscriber that will send an email if needed
+ *
+ * @author Wouter Sioen <wouter@sumocoders.be>
+ */
+class FormBuilderSubmittedMailSubscriber
+{
+    protected $mailer;
+
+    /**
+     * @param Swift_Mailer $mailer
+     */
+    public function __construct(Swift_Mailer $mailer)
+    {
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * @param FormBuilderSubmittedEvent $event
+     */
+    public function onFormSubmitted(FormBuilderSubmittedEvent $event)
+    {
+        $form = $event->getForm();
+
+        // need to send mail
+        if ($form['method'] == 'database_email') {
+            // build our message
+            $from = FrontendModel::getModuleSetting('Core', 'mailer_from');
+            $message = \Common\Mailer\Message::newInstance(sprintf(
```

Comment:
I think creating the message instance is in fact the responsibility of a service that is sending an email. The mailer itself is injected using constructor injection though.